library/src/main/java/kwasm/format/text/Type.kt

```diff
@@ -135,13 +138,17 @@ sealed class Type<T>(
         }
     }
 
+    /**
+     * From https://webassembly.github.io/spec/core/text/types.html#memory-types:
```

Comment:
Can you make this a link, like in the other KDocs?

Reply:
Done